features/support/env.rb

```diff
@@ -73,7 +73,10 @@ def initialize_environment
 
 After do
   if @last_run_result && !@error_expected
-    expect(@last_run_result.error).to be_falsy, 'Expected no runtime error'
+    if @last_run_result.error
+      puts unformatted_last_run_output
+      expect(@last_run_result.error).to be_falsy, 'Expected no runtime error'
+    end
```

Comment:
I think this would show the output in red, like if the output is an error. I prefer to keep the error message focussed on what exactly is wrong, and then just dump the output as a convenience for the developer. I'm not feeling strong about this though, so if you really want to do this, I'm open to it.